### DISCUSSION TITLE
整了一个用透明像素填充卡池图像的脚本

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#623] [General]
https://github.com/Mark9804/adachi-resource-assistant

@Xm798 抽卡用的图像从内鬼网上扒下来之后先过一遍这个脚本，然后再传上来

**Comment by Xm798** [comment]
妙啊

**Comment by mark9804** [comment]
现在可以自动识别是证件照还是抽卡图片了

**Comment by mark9804** [comment]
我打算拓展业务了，仓库改了个名字
https://github.com/Mark9804/adachi-resource-assistant

最终的设想是除了角色半身照只能手动裁剪之外，其他都可以直接获得可用的图像，最近打算把角色卡池文件全部替换掉，处理两个完全不同的版本的资源太麻烦了，限制我进一步发挥，尝试一下全部替换成内鬼网的版本可不可行

**Reply by mark9804** [reply]
@Xm798 我给你发了个合作邀请，因为人物和武器的头像得想办法去 https://ambr.top/chs/archive/avatar 那里爬，需要解析 xml ，我不太会这个

**Reply by Xm798** [reply]
> @Xm798 我给你发了个合作邀请，因为人物和武器的头像得想办法去 https://ambr.top/chs/archive/avatar 那里爬，需要解析 xml ，我不太会这个

好家伙 我瞅瞅 （我也不一定会哈哈哈哈哈

**Reply by Arondight** [reply]
最终的设想是用那个又笨又大的爬虫把图片也给扒齐了，有时间我想尽量不去用 puppeteer 了，真滴慢，不解析的话还不如直接发个请求

**Reply by Arondight** [reply]
不过爬网站过多的话就得应对多个页面可能的改版了，似乎以后维护爬虫也成了版本更新中可能的大活

**Reply by mark9804** [reply]
> 不解析的话还不如直接发个请求

我就是直接发请求的，不过不是因为慢，单纯是因为菜

> 不过爬网站过多的话就得应对多个页面可能的改版了，似乎以后维护爬虫也成了版本更新中可能的大活

从能节省下来的时间来看，这么做还是有必要的

**Reply by Arondight** [reply]
> 从能节省下来的时间来看，这么做还是有必要的

本来没打算做成一个批量更新的东西，设想中每个半月执行两三次就好了，事实上我们用的远比想象中频繁
